adafruit_usb_descriptor/util.py

```diff
@@ -46,3 +46,17 @@ def join_interfaces(*args):
                                                endpoint_address)
             base_endpoint_number = max_endpoint_address + 1
     return interfaces
+
+def renumbers_interfaces(*args):
```

Comment:
This shares a lot of code with the helper above. How about adding a kwarg instead that disables endpoint renumbering? It could be:
```python
def join_interfaces(*args, *, renumber_endpoints=True):
```

Then after setting the interface numb you can do:
```python
if not renumber_endpoints:
    continue
```

Reply:
Oh! If renumber_endpoints is False it should instead validate that no endpoint is set to 0. (Or is that ok on your chip?)

Reply:
Done! I have added validation that checks if endpoint addresses are different from 0.